src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -29,25 +37,80 @@ protected function configure()
      * @return void
      */
     protected function execute(InputInterface $input, OutputInterface $output): void
+    {
+        $sprykName = current((array)$input->getArgument(static::ARGUMENT_SPRYK));
+        if ($sprykName !== false) {
+            $this->dumpSprykOptions($output, $sprykName);
+
+            return;
+        }
+
+        $this->dumpAllSpryks($output);
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return void
+     */
+    protected function dumpAllSpryks(OutputInterface $output): void
     {
         $sprykDefinitions = $this->getFacade()->getSprykDefinitions();
-        $sprykDefinitions = $this->formatForTable($sprykDefinitions);
+        $sprykDefinitions = $this->formatSpryksTable($sprykDefinitions);
 
         $output->writeln('List of all Spryk definitions:');
+        $this->printTable($output, ['Spryk name', 'Description'], $sprykDefinitions);
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     * @param string $sprykName
+     *
+     * @return void
+     */
+    protected function dumpSprykOptions(OutputInterface $output, string $sprykName): void
+    {
+        $sprykDefinition = $this->getFacade()->getSprykDefinition($sprykName);
+        $tableRows = $this->formatOptionsTable($sprykDefinition['arguments']);
```

Comment:
I think it would be better to use `formatOptions` as you do not format a table here.